databaseGenerator.js

```diff
@@ -2,141 +2,157 @@ const fs = require('fs');
 const path = require('path');
 const faker = require('faker');
 
-var filePath = path.join(__dirname, `./schema.sql`);
-
-// INSERT INTO users (ID, NAME, TOWNLOC, FRIENDS, REVIEWS, PHOTOS) 
-//   VALUES ("LKAJASOI4F24902V4MVIOWE", "Susie Q", "Oakland, CA", 27, 99, 20);
-var numUsers = 2000;
-var numBusinesses = 100;
-var numReviews = 4000;
-var numPhotos = 1000;
-var numMessages = 100;
+const filePath = path.join(__dirname, './schema.sql');
+const numUsers = 300;
+const numBusinesses = 100;
+const numReviews = 4000;
+const numPhotos = 1000;
+const numMessages = 100;
+const photoToUserIds = [];
 
 module.exports.numPhotos = numPhotos;
-//========================================================
+
 // USERS SECTION
-var createUser = function(index) {
-  var user = {};
+const createUser = (index) => {
+  const user = {};
   user.id = index;
   user.name = faker.name.findName();
   user.townLoc = `${faker.address.city()}, ${faker.address.stateAbbr()}`;
   user.friends = Math.floor(Math.random() * 1000);
   user.reviews = Math.floor(Math.random() * 1000);
   user.photos = Math.floor(Math.random() * 100);
-  user.photo_id = 'NULL';
+  let photoId = Math.floor(Math.random() * numPhotos + 1);
+  while (photoToUserIds.includes(photoId)) {
+    photoId = Math.floor(Math.random() * numPhotos + 1);
+  }
+  photoToUserIds.push([photoId, user.id]);
+  user.photo_id = photoId.toString();
   return user;
-}
-var convertUsersToSql = function(users) {
-  var sql = [];
-  for (var i = 0; i < users.length; i++) {
-    sql[i] = `INSERT INTO users (ID, NAME, TOWNLOC, FRIENDS, REVIEWS, PHOTOS, PHOTO_ID) VALUES ("${users[i].id}", "${users[i].name}", "${users[i].townLoc}", ${users[i].friends}, ${users[i].reviews}, ${users[i].photos}, "${users[i].photo_id}");`
+};
+const convertUsersToSql = (users) => {
+  let sql = [];
+  for (let i = 0; i < users.length; i += 1) {
+    sql[i] = `INSERT INTO users (ID, NAME, TOWNLOC, FRIENDS, REVIEWS, PHOTOS, PHOTO_ID)
+    VALUES ("${users[i].id}", "${users[i].name}", "${users[i].townLoc}", ${users[i].friends}, ${users[i].reviews}, ${users[i].photos}, "${users[i].photo_id}");`;
   }
   sql = sql.join('\n');
   return sql;
-}
-var users = [];
-for (var i = 1; i <= numUsers; i++) {
+};
+let users = [];
+for (let i = 1; i <= numUsers; i += 1) {
   users.push(createUser(i));
 }
```

Comment:
Might be a personal preference, but I might consider encapsulating something like this (line 41 to 44) into a function (createUsersArray) that would return the array.
Similar on lines 62, 90, 125, 148.
Non-blocking.

Reply:
I might do that later.